src/app/core/config.py

```diff
@@ -19,6 +23,8 @@
 ELASTIC_TOTAL_SIZE = 500_000  # Maximum number of entries elasticsearch queries, very large to query all entries
 ELASTICSEARCH_TIMEOUT = int(os.getenv("ELASTICSEARCH_TIMEOUT", 20))
 
-BACKGROUND_TASK_TIME_INTERVAL = 10 * 60  # Time between consecutive background calls
+# Time in seconds between consecutive background calls
+BACKGROUND_TASK_TIME_INTERVAL = int(os.getenv("BACKGROUND_TASK_TIME_INTERVAL", 10 * 60))
 
-ENABLE_DATABASE = True
+# Whether to enable the database functionality for storing and loading historic quality matrix snapshots
+ENABLE_DATABASE = os.getenv("ENABLE_DATABASE", "True").lower() == "true"
```

Comment:
If this is false, we should also deactivate some of the endpoints, but for now i think its nice to be able to quickly disable it on a local system.